tests/backend_test.py

```diff
@@ -37,7 +45,9 @@ def test_ionq_simulator(azure_backend: AzureBackend) -> None:
         counts = r.get_counts()
         assert counts == Counter({(0, 0): 5, (1, 1): 5})
     else:
-        warn("ionq.simulator unavailable or queue time >= 60s: not submitting")
```

Comment:
I found it really tricky to debug this with only getting warnings. If you want to, I can move back to the warnings, but I think as long as we are in the testing stage, it would be better to keep this as an error.